docs/api/securing-api.md

```diff
@@ -49,7 +65,7 @@ Now let us go down the list!
 
 ### Create a new FormsApp Object
```

Comment:
`FormsApps.verifyJWT()` is a static function. It does not require an instance of `FormsApps` or a developer key.